doc/changelog.rst

```diff
@@ -6,6 +6,15 @@ Changelog
 Motor 2.4
 ---------
 
+Motor 2.4 adds support for client-side field-level encryption
```

Comment:
We should link to the newly added CSFLE examples here like we do in this changeling entry for PyMongo: https://pymongo.readthedocs.io/en/stable/changelog.html#changes-in-version-3-10-0